.travis.yml

```diff
@@ -19,4 +19,4 @@ before_install:
   - cd ${TRAVIS_BUILD_DIR}
 
 script:
-  - node_modules/.bin/gulp travis
+  - node_modules/.bin/gulp test
```

Comment:
@paulkerschen - I think node version is in `.travis.yml`. For other projects, when I updated node I added:
```
before_install:
  - nvm install 12
```
Not sure if something like that is necessary for certain versions of Node.

Reply:
I’ll update the version in travis.yml and see if that works.